test/functional/interface_rest.py

```diff
@@ -312,6 +312,16 @@ def run_test(self):
                             if 'coinbase' not in tx['vin'][0]}
         assert_equal(non_coinbase_txs, set(txs))
 
+        # Verify that the non-coinbase tx has "prevout" field set
```

Comment:
nit:
```suggestion
        # Verify that the non-coinbase tx has "prevout" key set
```
A Python dictionary is a set of key-value pairs.